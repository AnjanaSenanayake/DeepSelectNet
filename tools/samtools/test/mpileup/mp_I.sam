@SQ	SN:z	LN:13
@CO	Various I combinations
@CO	Starting with I isn't handled due to the pileup ordering.
@CO	The record only appears in the pileup list when the first base
@CO	is placed against the reference, which is too late.
@CO	T  AGCT   TAGCAGG  T ref
@CO	   AGCT   TAGCAGG
@CO	   AGCTCCCTAGCAGG
@CO	 A=AGCT   TAGCAGG=A
@CO	 AB0123ABC4567890CD
s1	0	z	2	0	11M	*	0	0	AGCTTAGCAGG	01234567890
s2	0	z	2	0	4M3I7M	*	0	0	AGCTCCCTAGCAGG	0123ABC4567890
s1	0	z	2	0	2I11M2I	*	0	0	A=AGCTTAGCAGG=A	AB01234567890CD
