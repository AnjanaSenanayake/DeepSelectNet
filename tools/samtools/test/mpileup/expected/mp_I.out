z	2	A	3	^!.^!.^!.	000
z	3	G	3	...	111
z	4	C	3	...	222
z	5	T	3	..+3CCC.	333
z	6	T	3	...	444
z	7	A	3	...	555
z	8	G	3	...	666
z	9	C	3	...	777
z	10	A	3	...	888
z	11	G	3	...	999
z	12	G	3	.$.$.+2=A$	000
